spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+describe PasswordResetsController do
+  describe "GET show" do
+    it "renders show template if token is valid" do
+      alice = Fabricate(:user)
+      alice.update_column(:token, '123')
+      alice.save
```

Comment:
Why do you `.save` - the `.update_column` should be enough, and you can as well just straight create alice with token. `alice = Fabricate(:user, token: '123')`